tests/test_preview.py

```diff
@@ -952,6 +1014,200 @@ def test_preview_colocated_db_script_ensemble(fileutils, test_dir, wlmutils, mlu
+        "cnn",
+        "TF",
+        model_path=model_path,
+        device=test_device,
+        devices_per_node=test_num_gpus,
+        first_device=0,
+        inputs="args_0",
+        outputs="Identity",
+    )
+
+    # Colocate a database with the first ensemble members
+    for i, entity in enumerate(colo_ensemble):
+        entity.colocate_db_tcp(
+            port=test_port + i, db_cpus=1, debug=True, ifname=test_interface
+        )
+        # Add ML models to each ensemble member to make sure they
+        # do not conflict with other ML models
+        entity.add_ml_model(
+            "cnn2",
+            "TF",
+            model_path=model_path,
+            device=test_device,
+            devices_per_node=test_num_gpus,
+            first_device=0,
+            inputs="args_0",
+            outputs="Identity",
+        )
+        entity.disable_key_prefixing()
+
+    # Add another ensemble member
+    colo_ensemble.add_model(colo_model)
+
+    # Colocate a database with the new ensemble member
+    colo_model.colocate_db_tcp(
+        port=test_port + len(colo_ensemble) - 1,
+        db_cpus=1,
+        debug=True,
+        ifname=test_interface,
+    )
+    # Add a ML model to the new ensemble member
+    model_inputs = "args_0"
+    model_outputs = "Identity"
+    model_name = "cnn2"
+    model_backend = "TF"
+    colo_model.add_ml_model(
+        model_name,
+        model_backend,
+        model_path=model_path,
+        device=test_device,
+        devices_per_node=test_num_gpus,
+        first_device=0,
+        inputs=model_inputs,
+        outputs=model_outputs,
+    )
+
+    exp.generate(colo_ensemble)
+
+    preview_manifest = Manifest(colo_ensemble)
+
+    # Execute preview method
+    output = previewrenderer.render(exp, preview_manifest, verbosity_level="info")
+
+    assert "Outgoing Key Collision Prevention (Key Prefixing)" not in output
+    assert "Devices Per Node" not in output
+
+
+def test_verbosity_level_info_orchestrator_preview_render(
+    test_dir, wlmutils, choose_host
+):
+    """Test correct preview output properties for Orchestrator preview"""
+    # Prepare entities
+    test_launcher = wlmutils.get_test_launcher()
+    test_interface = wlmutils.get_test_interface()
+    test_port = wlmutils.get_test_port()
+    exp_name = "test_orchestrator_preview_properties"
+    exp = Experiment(exp_name, exp_path=test_dir, launcher=test_launcher)
+    # create regular database
+    orc = exp.create_database(
+        port=test_port,
+        interface=test_interface,
+        hosts=choose_host(wlmutils),
+    )
+    preview_manifest = Manifest(orc)
+
+    # Execute method for template rendering
+    output = previewrenderer.render(exp, preview_manifest, verbosity_level="info")
+
+    # Evaluate output
+    assert "TCP/IP Port(s)" not in output
+    assert "Executable" not in output
+    assert "Run Command" not in output
+
+
+def test_preview_verbosity_info_ensemble(test_dir, wlmutils):
```

Comment:
Don't forget that you can write real unit tests that test the feature only and don't have to include the whole stack... 

Here, you could test verbosity more directly by using `monkeypatch` to get a path to a smaller template that is simpler, such as:

```
{%- if verbosity_level == Verbosity.INFO %}
INFO!
{%endif%}
{%- if verbosity_level == Verbosity.WARN %}
WARN!
{%endif%}
{%- if verbosity_level == Verbosity.DEBUG %}
DEBUG!
{%endif%}
```

Using that would focus a test only on something that can verify that your use of verbosity was correct and that your env.globals works as expected. And, it's easy to parametrize w/all 3 all input and expected outputs.

Ditto on you

Reply:
I like this a lot better